lib/linter/linter.js

```diff
@@ -2081,6 +2160,14 @@ class Linter {
         return internalSlotsMap.get(this).lastSourceCode;
     }
 
+    getTimes() {
+        return internalSlotsMap.get(this).times ?? { passes: [] };
+    }
+
+    getFixPasses() {
```

Comment:
Could we call this `getFixPassCount()` to indicate that this is a number rather than an array?